CDP4IME/Shell.xaml

```diff
@@ -121,31 +121,17 @@
 
                         <dxb:BarItemLinkSeparator />
 
-                        <dxb:BarSplitButtonItem Name="sbSave"
-                                                Content="Save"
+                        <dxb:BarButtonItem Name="sbSave"
+                                                Content="Export Session"
                                                 Glyph="{dxc:DXImage Image=Save_16x16.png}"
+                                                Description="Export a Session"
+                                                Hint="Export a Session to a compatible file-based format."
```

Comment:
"Export session data to a E-TM-10-25 Archive"

Reply:
The intent is to have more File based dals to choose from so maybe it might not always be an archive?